content/guides/models/app/features/panels/line-plot/smoothing.md

```diff
@@ -8,22 +8,30 @@ title: Smooth line plots
 weight: 30
 ---
 
-W&B supports three types of smoothing:
+W&B supports three types of back-end smoothing applied by W&B Server:
 
-- [exponential moving average]({{< relref "smoothing.md#exponential-moving-average-default" >}}) (default)
-- [gaussian smoothing]({{< relref "smoothing.md#gaussian-smoothing" >}})
-- [running average]({{< relref "smoothing.md#running-average" >}})
-- [exponential moving average - Tensorboard]({{< relref "smoothing.md#exponential-moving-average-deprecated" >}}) (deprecated)
+- [Time weighted exponential moving average (default)]({{< relref "#time-weighted-exponential-moving-average-default" >}}) 
+- [gaussian smoothing]({{< relref "#gaussian-smoothing" >}})
+- [running average]({{< relref "#running-average" >}})
+- [Tensorboard's exponential moving average smoothing algorithm]({{< relref "smoothing.md#exponential-moving-average" >}})
 
 See these live in an [interactive W&B report](https://wandb.ai/carey/smoothing-example/reports/W-B-Smoothing-Features--Vmlldzo1MzY3OTc).
 
 {{< img src="/images/app_ui/beamer_smoothing.gif" alt="" >}}
 
-## Exponential Moving Average (Default)
 
-Exponential smoothing is a technique for smoothing time series data by exponentially decaying the weight of previous points. The range is 0 to 1. See [Exponential Smoothing](https://www.wikiwand.com/en/Exponential_smoothing) for background. There is a de-bias term added so that early values in the time series are not biased towards zero.
 
-The EMA algorithm takes the density of points on the line (the number of `y` values per unit of range on x-axis) into account. This allows consistent smoothing when displaying multiple lines with different characteristics simultaneously.
+## Time Weighted Exponential Moving Average (Default)
+
+The Time Weighted Exponential Moving Average (TWEMA) smoothing algorithm is a technique for smoothing time series data by exponentially decaying the weight of previous points. The range is 0 to 1. See [Exponential Smoothing](https://www.wikiwand.com/en/Exponential_smoothing) for background. There is a de-bias term added so that early values in the time series are not biased towards zero.
+
+The TWEMA algorithm takes the density of points on the line (the number of `y` values per unit of range on x-axis) into account. This allows consistent smoothing when displaying multiple lines with different characteristics simultaneously.
+
+TWEMA smoothing is applied by W&B Server to a full scan of history, rather than sampling or bucketing first before smoothing. In general, it is more accurate than EMA smoothing. However, in the following situations, TWEMA smoothing is not supported and EMA smoothing is used instead:
+- Sampling
+- Grouping
+- non-monotonic x-axes
```

Comment:
This only applies to the exponential moving average smoothing algorithm not TWEMA

Reply:
Done, PTAL